docs/content/latest/yugabyte-platform/manage-deployments/high-availability.md

```diff
@@ -0,0 +1,183 @@
+
+Platform's high availability feature is an active-standby model for multiple platforms in a cluster with asynchronous replication. Your platform data is replicated across multiple VMs, ensuring that you can recover smoothly and quickly from a VM failure and continue to manage and monitor your universes, with your configuration and metrics data intact.
+
+## General Architecture
+
+Each HA cluster includes a single _active platform_ and at least one _standby platform_, configured as follows:
+
+* **The active platform** runs normally, but also pushes out backups of it’s state to all of the standby platforms in the HA cluster.
+
+* **A standby platform** is completely passive while in standby mode and can't be used for managing or monitoring clusters until you manually promote it to active.
+
+**Backups** from the active platform are periodically taken and pushed to followers at a user-configurable frequency (no more than once per minute). The active platform also creates and sends one-off backups to standby platforms whenever a task completes (such as creating a new universe). Metrics are duplicated to standby platforms using Prometheus federation. Standby platforms retain the 10 most recent backups on disk.
+
+When you promote a standby platform to active, Yugabyte Platform restores your selected backup, and automatically demotes the previous active platform to standby mode.
+
+## Setting up an HA Cluster
+
+### Prerequisites
+
+* Yugabyte Platform v2.5.3.1 or above.
+* You have already [installed](../../install-yugabyte-platform/) multiple YB Platform instances to be used in the HA cluster.
+* Platform VMs are all able to connect to each other over the port that the Platform UI/API is normally reachable over (80/443, for example).
+* All platforms are running the same version of Yugabyte Platform software (it is good practice to upgrade all platforms in the HA cluster at close to the same time).
+
+### Set up the Active Platform
+
+1. Click the "Admin" tab at the bottom of the left-side navigation panel.
+
+    <br/>
+
+    ![Replication configuration tab](/images/yp/high-availability/replication-configuration.png)
+
+1. Select the `Active` instance type.
+
+    <br/>
+
+    ![Active instance type](/images/yp/high-availability/instance-type-active.png)
+
+1. Enter this platform’s IP address or hostname (including the HTTP or HTTPS protocol prefix).
+
+    <br/>
+
+    ![Add an instance](/images/yp/high-availability/add-active-instance.png)
+
+1. Generate a key.
+
+    <br/>
+
+    ![Generate a shared key](/images/yp/high-availability/generate-key.png)
+
+1. Select your desired replication frequency, in minutes.
+
+    <br/>
+
+    In most cases, you don't need to replicate very often. Yugabyte recommends... **DANIEL, WHAT IS THE REC HERE?**
```

Comment:
maybe every 5 to 10 minutes? There really isn't any issue with setting it to 1m though (and this might give a better UX as opposed to a longer delay).